web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -241,9 +250,32 @@
     class="bg-white sticky left-0 top-0 overflow-visible"
     style="z-index:101"
   >
-    <div style:height="20px" style:padding-left="24px" />
     <!-- top axis element -->
     <div />
+    <div bind:this={axisTop} style:height="24px" style:padding-left="24px">
+      {#if $dashboardStore?.selectedScrubRange?.end && !$dashboardStore?.selectedScrubRange?.isScrubbing}
+        <Portal>
+          <FloatingElement
+            target={axisTop}
+            location="top"
+            relationship="direct"
+            alignment="middle"
+            distance={10}
+            pad={0}
+          >
+            <div style:left="-40px" class="absolute flex justify-center">
+              <Button compact type="highlighted" on:click={() => zoomScrub()}>
```

Comment:
"highlighted" is a bit confusing to me. This looks like a "secondary" button, or "secondary-alt" or something.

I see there's a large shadow on the button. Maybe that CSS belongs on the absolute positioning div?